src/components/Sidenav/Sidenav.style.ts

```diff
@@ -77,7 +77,8 @@ export const ButtonGroup = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: flex-end;
-  button {
+  button:not(:first-child),
+  a:not(:first-child) {
```

Comment:
I was fixing this yesterday.  If you leave this, we will have a warning in console:
 
![image](https://user-images.githubusercontent.com/51168865/113309092-16b5cf00-9307-11eb-9099-0723ce97311d.png)

There is no button here right now. Just add 
```suggestion
  a:last-of-type {
```

Reply:
made it  `a:not(:first-of-type) {` so it works even if we add more links in there